File: leetcode/algorithms/521_longest_uncommon_subsequence_1/main.dart
import 'dart:math';

class LongestUncommonSubsequenceI {
  int findLUSlength(String a, String b) {
    if (a == b) {
      return -1;
    }
    return max(a.length, b.length);
  }

  // Solution
  int solution(String a, String b) {
    return a == b ? -1 : max(a.length, b.length);
  }
}

File: leetcode/algorithms/521_longest_uncommon_subsequence_1/main.py
class LongestUncommonSubsequenceI:
    def findLUSlength(self, a: str, b: str) -> int:
        return -1 if a == b else max(len(a), len(b))


    # Solution
    def solution(self, a: str, b: str) -> int:
        return -1 if a == b else max(len(a), len(b))
